deployment/deployment_utils.sh

```diff
@@ -56,4 +56,22 @@ build_directories () {
   cp -r ./dataland-keycloak/realms "$target_dir"/dataland-keycloak
 
   cp ./deployment/{initialize_keycloak,migrate_keycloak_users,deployment_utils}.sh "$target_dir"/dataland-keycloak
-}
\ No newline at end of file
+}
+
+are_docker_containers_healthy_remote () {
```

Comment:
a) Das ist wieder ein `get_unhealthy_services` wie in dem lokalen test auch
b) das ist vom Code ziemlich identisch mit dem lokalen Test. Können wir das zusammenlegen?

Reply:
Jup. Zumindest über einen leichten umweg.

Reply:
ok